src/librustdoc/passes/collect_intra_doc_links.rs

```diff
@@ -1700,6 +1754,17 @@ fn resolution_failure(
                         item(res),
                         variant
                     ),
+                    ResolutionFailure::MalformedGenerics(variant) => match variant {
+                        MalformedGenericsVariant::UnbalancedAngleBrackets => {
+                            String::from("unbalanced angle brackets")
+                        }
+                        MalformedGenericsVariant::MissingType => {
+                            String::from("missing type for generic parameters")
+                        }
+                        MalformedGenericsVariant::HasUfcs => {
+                            String::from("UFCS syntax is unsupported")
```

Comment:
I guess this is kind of redundant since IIUC "UFCS" stands for "uniform functional call syntax", so this is "uniform function call syntax syntax", but I think it sounds clearer this way even if it's a bit redundant/verbose.

Reply:
I don't think we should print an abbreviation in the error message. Do you know what the reference calls this?

Reply:
It looks like the reference actually calls it "fully-qualified syntax" - I don't think I realized those were the same thing:

> ### Disambiguating Function Calls
>
> Rust treats all function calls as sugar for a more explicit, fully-qualified syntax. Upon compilation, Rust will desugar all function calls into the explicit form. Rust may sometimes require you to qualify function calls with trait, depending on the ambiguity of a call in light of in-scope items.
>
> > Note: In the past, the Rust community used the terms "Unambiguous Function Call Syntax", "Universal Function Call Syntax", or "UFCS", in documentation, issues, RFCs, and other community writings. However, the term lacks descriptive power and potentially confuses the issue at hand. We mention it here for searchability's sake.